local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -25,6 +25,10 @@ module.exports = [
     parse: (val) => val === 'false' ? false : true,
     default: true,
   }, {
+    command: '--minify [boolean]',
+    description: 'Enables or disables minification, overriding "--dev"',
```

Comment:
'overriding "--dev"'? May be better something like '(default: false for development build, otherwise true)'
Also "--dev" option description needs to be clarified...